sunpy/net/solarnet/tests/test_solarnet.py

```diff
@@ -0,0 +1,80 @@
+from pathlib import Path
+
+import pytest
+from parfive import Results
+
+import astropy.units as u
+
+import sunpy.net.attrs as a
+from sunpy.net import Fido
+from sunpy.net.base_client import QueryResponseTable
+from sunpy.net.solarnet import SOLARNETClient
+
+
+@pytest.fixture
+def client():
+    return SOLARNETClient()
+
+
+def test_info_url(client):
+    assert client.info_url == 'https://solarnet2.oma.be'
+
+
+@pytest.mark.remote_data
+def test_search():
+    query = [a.solarnet.Dataset.eui_level_2 , a.solarnet.Limit(2) , a.Detector("HRI_EUV")]
+    url = Fido.search(*query)
+    assert isinstance(url[0],QueryResponseTable)
+    assert len(url[0]) == 2
+    assert "/service/api/svo/metadata_eui_level_2/20200512T122556952/" in url[0]["Resource_uri"]
+
+
+def test_can_handle_query(client):
+    assert not client._can_handle_query(a.Time("2020/01/02", "2020/01/03"))
+    assert not client._can_handle_query(a.solarnet.Limit(10),a.solarnet.Target.ar)
+    assert client._can_handle_query(a.solarnet.Dataset.swap_level_1)
+
+
+def test_solarnet_attrs(client):
+    attrs = client.load_solarnet_values()
+    assert a.solarnet.Dataset in attrs.keys()
+    assert len(attrs[a.solarnet.Dataset]) > 0
+
+
+@pytest.mark.remote_data
+def test_fetch_return_type():
+    qr = Fido.search(a.solarnet.Dataset.swap_level_1 & a.solarnet.Limit(1) & a.solarnet.Tags.moon_transit)
+    res = Fido.fetch(qr)
+    assert isinstance(res, Results)
```

Comment:
Might be worth adding some more assertions here? Like len() == 1 if that's what we are expecting from `Limit` for example.